### PR TITLE
consensus: move test use only var/func to testfile

### PR DESCRIPTION
**Pull request by jsvisa**

### CODE REVIEW

File: consensus/clique/snapshot_test.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ type testerVote struct {
 	auth   bool
 }
 
+// blockPeriod is the default minimum difference between two consecutive block's timestamps
+var blockPeriod = uint64(15)
```
Comment (karalabe) [inline consensus/clique/snapshot_test.go]: I think we can get rid of this constant and directly inline `15` into the test that uses it. There's a single invocation, might as well clean out this altogether.

Comment (jsvisa) [reply consensus/clique/snapshot_test.go]: changed.

**Comment by motafa1320** [comment]
Hi
My email is 
dara_3326@yahoo.com
Pleas sending email me.

**Comment by jsvisa** [comment]
@karalabe Should it be appropriate to be merged this PR?

**Pull request merged**